### PR TITLE
Simplify `F.triplet` test

### PR DESCRIPTION
**Pull request by niboshi**
Applies the function test case https://github.com/chainer/chainer/pull/3499.

### CODE REVIEW

**Comment by kmaehashi** [comment]
Could you check conflicts?

Jenkins, test this please.

**Comment by hvy** [comment]
jenkins, test this please.

File: tests/chainer_tests/functions_tests/loss_tests/test_triplet.py [hunk 1]
```diff
@@ -18,131 +17,75 @@
...
-                continue
-            if (abs(self.a - self.n) < 2 * eps).any():
-                continue
-            dist = numpy.sum(
-                (self.a - self.p) ** 2 - (self.a - self.n) ** 2,
-                axis=1) + self.margin
-            if (abs(dist) < 2 * eps).any():
-                continue
-            break
+    def before_test(self, test_name):
```
Comment (asi1024) [inline tests/chainer_tests/functions_tests/loss_tests/test_triplet.py]: Remove it because #6069 is merged.

**Comment by niboshi** [comment]
Jenkins, test this please.

**Comment by niboshi** [comment]
Rebased

**Comment by niboshi** [comment]
This test is unstable around nondifferentiable points. Further investigation is required.

**Comment by hvy** [comment]
Just dropping a reference, in case we'd want to pass the test without using the non-differentiable-dodging. It can be closed otherwise. https://github.com/chainer/chainer/pull/6855

**Pull request closed without merging**